index.js

```diff
@@ -1,9 +1,12 @@
 var io = require('indian-ocean')
 
-var display = require('./nodebooks.js')
```

Comment:
this is tricky... using a local dependency is better for lots of reasons (e.g. third party viewer plugins), but a magical global makes instrumentation easier. not sure how this is ultimately best approached